style.css

```diff
@@ -657,3 +683,726 @@ ul li {
   background-color: #fff;
   color: black;
 }
+
+@media (min-width: 768px) {
+  body {
+    height: 5125px;
+    width: 1440px;
+    left: 0;
+    top: 0;
+    border-radius: 0;
+    position: relative;
+    background: #dfe1e6;
+  }
+
+  /* .container {
+    height: 916px;
+    width: 1440px;
+    left: 0;
+    top: 1px;
+    position: absolute;
+    background: #0d4ed1;
+    border-radius: 0 0 0 10%/0 0 0 10%;
+  } */
+  .container {
+    position: relative;
+    width: 1440px;
+    height: 5125px;
+    background: #f4f5f7;
+  }
```

Comment:
- [ ] Please make sure there is no empty space at the right side of the screen. Make sure the element takes the whole page for all sections even when the window is resized. 

**Your project**
![image](https://user-images.githubusercontent.com/74411135/236562930-21cc50f3-8e83-48e6-b442-514cc1eb3058.png)

![image](https://user-images.githubusercontent.com/74411135/236563162-589c4d59-e110-4552-b767-66e7f6ada252.png)

**figma design**
![image](https://user-images.githubusercontent.com/74411135/236563227-cff2b9ba-4c0d-48ae-9619-d1ca63f8c5a1.png)

![image](https://user-images.githubusercontent.com/74411135/236563349-8619eccd-b9da-4dac-b2cd-717d0c36df2f.png)

This will make your project way prettier which will improve the user experience 📱  😄 👍

Reply:
@Trast00   Hey man I have tried to fix this, but It won't work would you help?

Reply:
Done